demisto_sdk/commands/validate/file_validator.py

```diff
@@ -742,6 +742,18 @@ def run_all_validations_on_file(self, file_path: str, file_type: str = None) ->
             if not incident_type_validator.is_valid_incident_type(validate_rn=False):
                 self._is_valid = False
 
+        elif 'CHANGELOG' in file_path:
+            # since the changelog has not changed, we can ignore "no release notes" and "no new RN entry" errors
+            rn_ignore_errors = ignored_errors_list
+            file_name = file_path.split('/')[-1]
+            if file_name in rn_ignore_errors:
+                rn_ignore_errors[file_name].append("RN100", "RN101")
+
+            else:
+                rn_ignore_errors[file_name] = ["RN100", "RN101"]
```

Comment:
why?

Reply:
The all files validation treats all existing files as though they were added right now. This will result in these errors as it will not find any new entries in the changelog. Left a comment in the code with the same explanation.